webpack.config.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-undef */
```

Comment:
**Can this inline eslint rule be removed, please define eslint rule in the eslint file.**
I trust you can do this before merging no need of sending another CRR because of this 😄 .